### PR TITLE
[ADD] sale_department and invoice_department migrated to 8.0 version

### PR DESCRIPTION
**Pull request by omar7r**

### CODE REVIEW

File: invoice_department/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'Invoices with Department Categorization',
+    'version': '1.0',
+    'category': 'Generic Modules/Sales & Purchases',
+    'description': """\
+Add the department on Invoices as well as the related \
```
Comment (pedrobaeza) [inline invoice_department/__openerp__.py]: \ are not needed in a multi-line text.

**Comment by pedrobaeza** [comment]
Hi, Omar, thanks for the PR. You have removed files from `__unported__` and added outside, but it seems you have not used `git mv`, and thus the diff is not correct. Can you please check it?

**Comment by omar7r** [comment]
Hi Pedro,

I made git mv to rename files, in the commit you could see that some files are marked as renamed but others not, do you know if number of changes affects in this?
if not, I have to redo this PR

I just removed the \

**Comment by pedrobaeza** [comment]
If you have done it this way, maybe it's git that sometimes a lot of changes are taken as new files (maybe tabs<>spaces switch?), so don't worry, it's not so important.

You have my :+1:, but it would be good if you migrate it to new API :wink: 

Cheers.

**Comment by pedrobaeza** [comment]
Oops, a little PEP8 error:

```
./invoice_department/__init__.py:2:1: W391 blank line at end of file
```

Please fix it

**Comment by omar7r** [comment]
PEP8 fixed

**Comment by pedrobaeza** [comment]
Thanks.

**Comment by dreispt** [comment]
LGTM :+1:

**Pull request merged**